app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/ViewEventLogsFragmentPresenter.kt

```diff
@@ -18,7 +18,8 @@ import javax.inject.Inject
 class ViewEventLogsFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
-  private val viewModelProvider: ViewModelProvider<ViewEventLogsViewModel>
+  private val viewModelProvider: ViewModelProvider<ViewEventLogsViewModel>,
+  private val singleTypeAdapterFactory: BindableAdapter.SingleTypeBuilder.Factory
```

Comment:
For consistency: `singleTypeBuilderFactory`

Reply:
Updated.

Reply:
This appears addressed in the latest code. Marking as resolved since Rajat has already approved the PR.